src/Sylius/Bundle/ResourceBundle/DependencyInjection/Driver/Doctrine/DoctrinePHPCRDriver.php

```diff
@@ -49,6 +50,18 @@ protected function addRepository(ContainerBuilder $container, MetadataInterface
             $this->getClassMetadataDefinition($metadata),
         ]);
 
+        if ($metadata->hasParameter('translation')) {
+            $repositoryReflection = new \ReflectionClass($repositoryClass);
+            $translatableRepositoryInterface = TranslatableResourceRepositoryInterface::class;
+            $translationConfig = $metadata->getParameter('translation');
+
+            if (interface_exists($translatableRepositoryInterface) && $repositoryReflection->implementsInterface($translatableRepositoryInterface)) {
```

Comment:
I think you are right, but i copied this code from DoctrineORMDriver. I though it has to be the same.